com.civicrm.hrjobroles/CRM/Hrjobroles/Import/Form/DataSource.php

```diff
@@ -60,8 +58,12 @@ public function preProcess() {
   public function buildQuickForm() {
     //Setting Upload File Size
     $config = CRM_Core_Config::singleton();
-
-    $uploadFileSize = CRM_Core_Config_Defaults::formatUnitSize($config->maxFileSize.'m');
+    if ($config->maxImportFileSize >= 8388608) {
+      $uploadFileSize = 8388608;
+    }
+    else {
+      $uploadFileSize = $config->maxImportFileSize;
+    }
```

Comment:
Oh, that's true. I thought you would just fix this in a future commit/PR for 1.6. Good idea using the same fix for 1.5 and 1.6.

About the value, there are two things that bother me:
- It comes from nowhere. Why 8mb instead of 8.5, 7 or even 9? The way it's is it's just a random value. It could be anything
- It overrides the maxImportFileSize in $config. This is quite confusing. Imagine you've configure 10mb as the max import file, then you try to upload a 10mb file and will get an error. It'll probably take some time until you figure out this piece of code overriding the configured value.

If we can't find out what's this 8388608 I think we should always use the value in $config-maxImportFileSize